README.md

```diff
@@ -6,17 +6,17 @@
 
 [![Build Status](https://github.com/itsallcode/openfasttrrace-intellij-pluginactions/workflows/ci-build.yml/badge.svg)](https://github.com/itsallcode/openfasttrrace-intellij-pluginactions/workflows/ci-build.yml)
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aexasol-virtual-schema-lua&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.exasol%3Aexasol-virtual-schema-lua)
```

Comment:
Would it make sense or is it technically possible to use project-keeper, here?

Reply:
It's a gradle-based project unfortunately. PK does not support Gradle.